ambari-server/src/main/java/org/apache/ambari/server/controller/internal/HostComponentResourceProvider.java

```diff
@@ -120,6 +120,8 @@
   public static final String HOST_COMPONENT_MAINTENANCE_STATE_PROPERTY_ID
       = "HostRoles/maintenance_state";
   public static final String HOST_COMPONENT_UPGRADE_STATE_PROPERTY_ID = "HostRoles/upgrade_state";
+  public static final String HOST_COMPONENT_CLUSTER_ID
```

Comment:
We should not be adding service-specific properties to the ResourceProvider contract.